testsuite/pangu_lib/testnet_commands/commands.py

```diff
@@ -26,8 +27,8 @@
 @click.option("--layout-path", help="Pass the path to the layout file (yaml).")
 @click.option(
     "--framework-path",
-    default=f"{util.TEMPLATE_DIRECTORY}/framework.mrb",
-    help="Pass in the path to the compiled move framework (head.mrb, or framework.mrb) file. Defaults to the default framework in the pangu_lib.",
+    required=True,
```

Comment:
it'd be good to link to an example one, or instructions how to get one. I spent some time figuring out what this is

Reply:
added to the help message :) 

```
$ pangu testnet create --help           
Usage: pangu.py testnet create [OPTIONS]

  Create a testnet.

Options:
...
  --framework-path TEXT           Pass in the path to the compiled move framework (head.mrb, or framework.mrb) file. To compile it, run: $
                                  cargo run --locked --package aptos-framework -- release  [required]
```